adapter-http/src/main/resources/activities/baselines/http-rest-keyvalue.yaml

```diff
@@ -21,7 +21,7 @@ bindings:
   weighted_hosts: WeightedStrings('<<restapi_host:stargate>>')
 
   request_id: ToHashedUUID(); ToString();
-  request_token: ToString(); TextOfFile("TEMPLATE(stargate_tokenfile,data/stargate_token.txt)")
+  request_token: ToString(); FirstLines("TEMPLATE(stargate_tokenfile,data/stargate_token.txt)");
```

Comment:
So you have to write a token to a file in order to run this test? Should that be documented in the accompanying markdown file? I prefer the way things used to work where you pass the token as a property when invoking the script.

Reply:
@jeffreyscarpenter I agree, thanks for pointing this out.  The original intent was to perform  something similar to:  

`TEMPLATE(stargate_token, FirstLines(data/stargate_token.txt))`

I'll need to adjust some things to allow for this passing of the token with the externalized file option.  Good catch!

Also, as context, in the next iteration of the http activities (after we shake out issues) I was planning to obtain the token via http for the user to avoid having to supply a token at all.  However, it would still provide a direct submission as well.  wdyt?

I'll get the initial issue adjusted for this upcoming release.

Reply:
noted, merging this for preview release

Reply:
an option to get the token via http would be useful.